shimmingtoolbox/cli/prepare_fieldmap.py

```diff
@@ -24,7 +24,9 @@
               help="Output filename for the fieldmap, supported types : '.nii', '.nii.gz'")
 @click.option('-mask', 'fname_mask', type=click.Path(exists=True), help="Input path for a mask. Used for PRELUDE")
 @click.option('-threshold', 'threshold', type=float, help="Threshold for masking. Used for: PRELUDE")
-def prepare_fieldmap_cli(phase, fname_mag, unwrapper, fname_output, fname_mask, threshold):
+@click.option('-gaussian_filter', 'gaussian_filter', type=bool, help="Gaussian filter for B0 map")
+@click.option('-sigma', 'sigma', type=float, help="Standard deviation of gaussian filter. Used for: gaussian_filter")
```

Comment:
```suggestion
@click.option('-sigma', type=float, help="Standard deviation of gaussian filter. Used for: gaussian_filter")
```

FYI, if the name of the variable is the same as the name of the option, you don't need to specify it, it will automatically use that name.